components/utils/tooltip/CustomTooltip.module.css

```
@@ -0,0 +1,279 @@
+  transform-origin: center top;
+}
+
+.tooltip--left {
+  transform-origin: right center;
+}
+
+.tooltip--right {
+  transform-origin: left center;
+}
+
+@keyframes tooltipIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "tooltipIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)